### PR TITLE
handling kernels by name

### PR DESCRIPTION
**Pull request by mslacken**
version is also stored, kernel name defaults to its version

### CODE REVIEW

File: internal/app/wwctl/kernel/imprt/main.go [hunk 1]
```diff
@@ -14,58 +14,57 @@ import (
...
-			}
-		}
-		output, err := kernel.Build(arg, OptRoot)
-		if err != nil {
-			wwlog.Printf(wwlog.ERROR, "Failed building kernel: %s\n", err)
-			os.Exit(1)
+
+	// Checking if container flag was set, then overwriting OptRoot
+	kernelVersion := args[0]
+	kernelName := args[1]
```
Comment (gmkurtzer) [inline internal/app/wwctl/kernel/imprt/main.go:20]: This is going to cause a panic if args[1] is undefined.

**Comment by gmkurtzer** [comment]
I started doing a review on this, but I think there are some areas that will cause some pretty major issues, and if I am understanding the intention behind this properly, I believe this can be done much simpler. 

Pushing this off until milestone 4.3.0.

File: internal/app/wwctl/kernel/imprt/main.go (hunk 1 above)
Comment (mslacken) [reply internal/app/wwctl/kernel/imprt/main.go]: Should be fixed now.

**Comment by mslacken** [comment]
I don't see were major issues can occur, as the kernels and their drivers are stored by name, which can with this patch set to an arbitrary name. If no name is given, the import falls back to the old behavior.

**Comment by gmkurtzer** [comment]
Is the goal here just to be able to use an arbitrary kernel binary but still use the modules at `/var/modules/$(uname -r)`?

**Pull request merged**